google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -336,6 +336,9 @@ clouddebug.runconfig.formoredetails=For more details,
 clouddebug.runconfig.documentation.url=https://cloud.google.com/debugger/
 clouddebug.runconfig.documentation.url.text=click here
 
+gcs.panel.toolbar.refresh.hover.text=Refresh
```

Comment:
instead of having a refresh button could we listen for changes?
https://cloud.google.com/storage/docs/pubsub-notifications